Misc/NEWS.d/next/Library/2020-07-06-16-58-53.bpo-41207.Emw7Nk.rst

```diff
@@ -0,0 +1 @@
+In distutils.spawn, rewrite FileNotFound as DistutilsExecError.
```

Comment:
I'm not sure why you only mention FileNotFoundError (you forgot "Error" ;-)). Your change catchs any OSError, not only FileNotFoundError.

Reply:
In a2faa87, I've removed `FileNotFound` from the blurb. I mentioned `FileNotFound[Error]` because that was the exception that was unexpectedly encountered. In this latest blurb, I focus more on describing the user's expectation.